tests/smoke-bundler.yaml

```diff
@@ -27,12 +27,10 @@ input:
           type: git_source
           username: x-access-token
 output:
-    - type: record_package_manager_version
+    - type: record_ecosystem_versions
       expect:
         data:
-            ecosystem: deprecated
-            package-managers:
-                bundler: "2"
+            ecosystem: ""
```

Comment:
This is wrong, this section shouldn't change, I'm not sure why this is happening...

Reply:
https://github.com/dependabot/cli/pull/147